cli/cmd/cloud_account.go

```diff
@@ -182,18 +184,34 @@ func buildDetailsTable(integration api.V2RawType) string {
 				for i, j := range val {
 					if v, ok := j.(string); ok {
 						details = append(details, []string{strings.ToUpper(format.SpaceUpperCase(i)), v})
+					} else {
+						cli.Log.Warn("unable to build table details, unknown type", "type", i, "key", k)
 					}
 				}
 			case []any:
 				var values []string
 				for _, i := range val {
-					values = append(values, i.(string))
+					if _, ok := i.(string); ok {
+						values = append(values, i.(string))
+					} else if _, ok := i.(map[string]interface{}); ok {
+						for m, n := range i.(map[string]interface{}) {
+							values = append(values, fmt.Sprintf("%s:%s", m, n))
+						}
+					} else {
+						cli.Log.Warn("unable to build table details, unknown type", "type", i, "key", k)
+					}
 				}
 				details = append(details, []string{strings.ToUpper(format.SpaceUpperCase(k)), strings.Join(values, ",")})
 			}
 		}
 	}
 
+	// get server token for container registry type only
+	if c, ok := integration.(api.ContainerRegistryRaw); ok {
+		details = append(details, []string{"SERVER_TOKEN", c.ServerToken.ServerToken})
```

Comment:
This code panic's in CI:
```
                Error:          Should be empty, but was panic: runtime error: invalid memory address or nil pointer dereference                    
                                [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xcaa29f]                                              
                                                                                                                                                    
                                goroutine 1 [running]:                                                                                              
                                github.com/lacework/go-sdk/cli/cmd.buildDetailsTable({0x1283738, 0xc000558100})                                     
                                        /codefresh/volume/go-sdk/cli/cmd/cloud_account.go:211 +0x15f                                                
                                github.com/lacework/go-sdk/cli/cmd.containerRegistryShow(0x18a58a0?, {0xc000466af0, 0x1, 0x1?})                     
                                        /codefresh/volume/go-sdk/cli/cmd/container_registry.go:127 +0x4b7                                           
                                github.com/spf13/cobra.(*Command).execute(0x18a58a0, {0xc000466ad0, 0x1, 0x1})                                      
                                        /codefresh/volume/go-sdk/vendor/github.com/spf13/cobra/command.go:856 +0x67c                                
                                github.com/spf13/cobra.(*Command).ExecuteC(0x18acba0)                                                               
                                        /codefresh/volume/go-sdk/vendor/github.com/spf13/cobra/command.go:974 +0x3b4                                
                                github.com/spf13/cobra.(*Command).Execute(...)                                                                      
                                        /codefresh/volume/go-sdk/vendor/github.com/spf13/cobra/command.go:902                                       
                                github.com/lacework/go-sdk/cli/cmd.Execute()                                                                        
                                        /codefresh/volume/go-sdk/cli/cmd/root.go:159 +0x111                                                         
                                main.main()                                                                                                         
                                        /codefresh/volume/go-sdk/cli/main.go:29 +0x19                                                               

```

Reply:
ah got it nil check on the ServerToken for all of the non inline/proxy scanner types.